lib/bot/cmds/thanks.js

```diff
@@ -21,37 +23,63 @@ const thanksCommands = {
 
     const mentions = input.message.model.mentions;
     // just 'thanks' in a message
-    if (mentions && mentions.length === 0) {
+    if (_.isEmpty(mentions)) {
       Utils.warn('thanks', 'without any mentions', input.message.model);
       return null;
```

Comment:
Use a consistent return value such as `''` (*empty string*) as primary return value is a type of `string`. *Please test it thoroughly so that consumer of this method handles this appropriately and there is no silent bugs*.

If returning `null`, then mention in the documentation of the method.